### DISCUSSION TITLE
You must add this app to your distrib !

### DISCUSSION THREAD

**Discussion by zachari043** [elementary/triage#248] [Ideas]
https://github.com/stsdc/monitor
![Captures d'écran de 2023-06-08 14 24 32](https://github.com/elementary/triage/assets/30619426/8150742a-ccd5-4edb-8994-d8b2fbc574ec)

**Comment by ghost** [comment]
This is better than most Task Managers I know. Is it in appcenter?

Personally I think this kind of app should be integrated in the OS, but elementary's simplicity is what really makes it good in my opinion.

**Reply by stsdc** [reply]
Thank you for kind words :blush:

**Comment by zachari043** [comment]
this in not in appcenter... must be added !  it have look and feel of the OS !

**Reply by ghost** [reply]
You should publish it there then! Very very polished

**Comment by RMcNeely** [comment]
Several years back Monitor was in AppCenter but if I remember correctly it doesn't work correctly with flatpak. I'm not sure if this is something that has ever changed or not (since it was something outside elementary and the developers control). I think there was an old issue on the GitHub repo about what it would need to be supported again.

**Comment by 1280px** [comment]
+1 for this, would be nice to see it pre-installed by default

**Comment by nicolasauthier** [comment]
Looks nice. But I use Mission Center that is in the App Center and it does the job for me.

**Comment by fayjies** [comment]
I would like to see it by default in the elementary os 8. Elementary os does not use background app notification icons, and some apps require killing (as they keep running when closed). Such an app comes in handy in such a situation. Also, the app has the feeling of the OS.

**Comment by adrien901** [comment]
I'm currently using this one and I think it's pretty decent too! (Yup sorry it's all in french but nobody is perfect)

![image](https://github.com/elementary/triage/assets/166648712/d5884216-6c95-4cd2-812f-7ac6e72fd43e)

Installed from flathub directly

![image](https://github.com/elementary/triage/assets/166648712/8ef4988a-637c-4a30-b84d-b10ef52c54e9)

I was very surprised too that this was not included by default in the OS, but really that's no big deal for me, gives us a chance to choose the one we prefer and talk about it!

**Comment by nicekiwi** [comment]
This app is **NOT** currently in the flathub app store. 

If @stsdc submits it to flathub, it will also appear in the elementaryOS app center. https://docs.flathub.org/docs/for-app-authors/submission/

**Reply by stsdc** [reply]
Monitor needs elementary runtime. Last time I checked, it wasn't available in Flathub.

**Comment by danirabbit** [comment]
Gonna close as completed since we shipped monitor in OS 8.1!

**Reply by 1280px** [reply]
Happy New Year
